### PR TITLE
Adding French to the list of available language

### PR DESCRIPTION
**Pull request by fguitton**
This Pull request focuses on bringing French as an available language in the admin UI as well as fixing an indentation inconsistency in language files

### CODE REVIEW

File: app/static/i18n/fr.json [hunk 1]
```diff
@@ -0,0 +1,147 @@
+{
+    "STATUSBAR": {
+        "CLUSTER": "Grappe",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: Je ne suis pas sûre que Grappe soit la bonne traduction ici. Je préfère garder Cluster, c'est beaucoup plus comprehensible.

File: app/static/i18n/fr.json [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        "VERSION": "Version",
+        "SNAPSHOT": "INSTANTANÉ",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: Pareil pour snapshot, je ne trouve pas que 'Instantané' explique bien le concept

File: app/static/i18n/fr.json [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        "NODES": "Nœuds",
+        "HEALTH": "Santé",
+        "DATA": "Données",
+        "CHECKS": "Contrôles",
+        "CHECK": "Contrôle",
+        "FAILED": "échec",
+        "CLUSTER_OFFLINE": "Grappe hors-ligne",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: meme commentaire ici

File: app/static/i18n/fr.json [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        "CONSOLE": "Console",
+        "TABLE": "Tables",
+        "VIEW": "Vues",
+        "CLUSTER": "Grappe"
+    },
+    "OVERVIEW": {
+        "HEALTH": "Santé",
+        "REPLICATED_DATA": "Données Répliquées",
+        "AVAILABLE_DATA": "Données Disponibles",
+        "TOTAL_RECORDS": "Nombre Total d'Entrée",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: "Nombre Total d'Entrées

File: app/static/i18n/fr.json [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+        "UPDATE_TIP": "Merci de considérer une mise à jour vers la verion {{ version }}!",
+        "NO_NOTIFICATION": "Oh no! Il n'y a pas encore de notification.",
+        "MARK_ALL_READ": "Marquer comme lu"
+    },
+    "NAVIGATION": {
+        "OVERVIEW": "Aperçu",
+        "CONSOLE": "Console",
+        "TABLE": "Tables",
+        "VIEW": "Vues",
+        "CLUSTER": "Grappe"
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: Cluster

File: app/static/i18n/fr.json [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+    },
+    "OVERVIEW": {
+        "HEALTH": "Santé",
+        "REPLICATED_DATA": "Données Répliquées",
+        "AVAILABLE_DATA": "Données Disponibles",
+        "TOTAL_RECORDS": "Nombre Total d'Entrée",
+        "UNDERREPL_RECORDS": "Entrées Sous-Répliquées",
+        "UNAVAIL_RECORDS": "Entrées Non-Disponible",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: Entrées Non-Disponibles

File: app/static/i18n/fr.json [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+        "CLUSTER_LOAD": "Charge de la Grappe",
+        "CLUSTER_CHECKS": "Contrôles de la Grappe",
+        "NODES_CHECKS": "Contrôles des Nœuds",
+        "DISMISS": "Ignorer",
+        "DISMISS_ALL": "Tout ignorer",
+        "AFFECTED_NODES": "Nœuds affectés",
+        "INFO": "INFO",
+        "WARNING": "ALARME",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: C'est plus `avertissement` que `alarme`

File: app/static/i18n/fr.json [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+        "NAME": "Nom",
+        "NODES": "Nœuds",
+        "OVERVIEW": "Aperçu",
+        "HEALTH": "Santé",
+        "HOSTNAME": "Nom d'hôte",
+        "CRATE_VERSION": "Version de CrateDB",
+        "SNAPSHOT": "instantané",
+        "HTTP_ENDPOINT": "Point de connection HTTP",
+        "CPU_USAGE": "Usage CPU",
+        "HEAP_USAGE": "Usage du tas",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: Je dirais plutôt `pile` au lieu de `tas`

File: app/static/i18n/fr.json [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+        "DISK_USAGE": "Usage Disque",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: Usage du disque

File: app/static/i18n/fr.json [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+        "LOAD": "Charge",
+        "MINUTES": "min",
+        "CPU_CORES": " CPU",
+        "CORES": "Cœurs",
```
Comment (m-kharbat) [inline app/static/i18n/fr.json]: Je ne sais pas si `Cœurs` est le terme informatique le plus approprié. `noyaux` peut être ?

**Comment by m-kharbat** [comment]
A Changes.txt entry is missing as well

File: app/static/i18n/fr.json (hunk 7 above)
Comment (m-kharbat) [reply app/static/i18n/fr.json]: `alerte` c'est plus approprié

File: app/static/i18n/fr.json (hunk 10 above)
Comment (m-kharbat) [reply app/static/i18n/fr.json]: Bon finalement ce serait plus juste de garder `Cœurs `

File: app/static/i18n/fr.json (hunk 8 above)
Comment (fguitton) [reply app/static/i18n/fr.json]: There is conceptual difference between `stack` and `heap` (`pile` et `tas`), hence the choice in this particular case.

File: app/static/i18n/fr.json (hunk 1 above)
Comment (fguitton) [reply app/static/i18n/fr.json]: I understand the comment, strictly speaking, `cluster` is an English word exclusively. The official recommendation of the Ministère de la Culture mandates the use of the word `grappe`. It's all up to you 😄 

You can see the details on the official [FranceTerm website](http://www.culture.fr/franceterme/result?francetermeSearchTerme=cluster&francetermeSearchDomaine=0&francetermeSearchSubmit=rechercher&action=search)

File: app/static/i18n/fr.json (hunk 2 above)
Comment (fguitton) [reply app/static/i18n/fr.json]: There are no official recommendation for the word `snapshot` however companies, the likes of Amazon of Microsoft routinely use `instantanés` in there official documentation and products.

Example here : https://docs.aws.amazon.com/fr_fr/AWSEC2/latest/UserGuide/ebs-creating-snapshot.html
Or there : https://docs.microsoft.com/fr-fr/virtualization/hyper-v-on-windows/user-guide/checkpoints

A simple Wikipedia-ing will get you the same usage:
https://fr.wikipedia.org/wiki/Instantané_(informatique)

To be more be specific some use `capture instantanée` to reflect both `snap` and `shot` separately.
That said, I hear that others, such as VMware, do not; and use the word `snapshot` instead.

File: app/static/i18n/fr.json (hunk 10 above)
Comment (fguitton) [reply app/static/i18n/fr.json]: Yes, `noyau` refers to the notion of `kernel` 😃

File: app/static/i18n/fr.json (hunk 1 above)
Comment (m-kharbat) [reply app/static/i18n/fr.json]: In the definition of the Ministère de la Culture the definition of grappe is `Ensemble d'appareils de même type (terminaux, ordinateurs, etc.) rattachés à une même unité de contrôle.` Here what we mean by Cluster is more a group of nodes connected to each other not to a control unit.

In any case, I consulted other french speaking developers. I would still keep Cluster and Snapshot even if `Grappe` and `Capture instantanée` are direct translations. It's definitely simpler to understand.

File: CHANGES.txt [hunk 11]
```diff
@@ -10,6 +10,8 @@ Unreleased
 
 - Fixed the navigation component to allow opening URLs in new tabs.
 
+- Addition of French language files and menu options.
```
Comment (m-kharbat) [inline CHANGES.txt]: This should be in the top of the changelog

**Comment by m-kharbat** [comment]
@fguitton please squash your commits before I can merge this pr

**Pull request merged**